classes/Parameters/UpgradeConfiguration.php

```diff
@@ -160,6 +184,20 @@ public function shouldUpdateRTLFiles(): bool
         return (bool) $this->get('PS_AUTOUP_UPDATE_RTL_FILES');
     }
 
+    public static function shouldDisabledOverride(): bool
+    {
+        return (bool) Configuration::get('PS_DISABLE_OVERRIDES');
+    }
+
+    public static function updateDisabledOverride(bool $value, ?int $shopId = null): void
+    {
+        if ($shopId) {
```

Comment:
It probably doesn't make sense to rely on `$shopId` in this module, you work on one PrestaShop instance, not on a single store.

Reply:
@kpodemski 

I took this configuration which was already present at the base. I tested to make sure I understand why this configuration was added. 

It is possible to set up this configuration in a particular shop in the advanced settings. 

When adding this configuration in the autoupgrade module, this feature with the context of a shop was also added, to keep the same value on this configuration between the 2 pages.

Reply:
Disabling overrides makes sense to try the shop after upgrade. Unfortunately the global value can be overriden if a specific shop has its own configuration value.

The current implementation covers this case.